app/controllers/plantings_controller.rb

```diff
@@ -31,7 +31,7 @@ def show
     @planting = Planting.includes(:owner, :crop, :garden, :photos)
       .friendly
       .find(params[:id])
-    @photos = @planting.photos.order(created_at: :desc).includes(:owner).paginate(page: params[:page])
+    @photos = @planting.photos.order(date_taken: :desc).includes(:owner).paginate(page: params[:page])
```

Comment:
Do we want to migrate old data (fall back on created at if null? Fetch from flickr again?)

Reply:
What's the performance hit of that?


ALSO, i just realised we'd want to index that column too - so the query can order it faster

Reply:
Updating existing records in a migration to fall back to created_at or re-lookup from flickr - one off, no future perf considerations

Using COALESCE... its fairly fast, particularly if we added the index on the coalesce key itself.